.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,137 @@
+---
+name: 🐛 Bug report
+description: Create a report to help us improve
+issue_body: false  # default: true, adds a classic WSYWIG textarea, if on
+
+body:
+- type: markdown
+  attributes:
+    value: |
+      ⚠
+      Verify first that your issue is not [already reported on GitHub][issue search].
+      Also test if the latest release and devel branch are affected too.
+      *Complete **all** sections as described, this form is processed automatically.*
+
+      [issue search]: https://github.com/ansible/ansible/search?q=is%3Aissue&type=issues
+
+
+- type: textarea
+  attributes:
+    label: SUMMARY
```

I've already check, and we can make this less shouty without breaking the bot
```suggestion
        label: Summary
```